kitsune/questions/views.py

```diff
@@ -556,82 +573,64 @@ def aaq(
 
                 return HttpResponseRedirect(path)
 
-    if request.method == "GET":
-        form = None
-        featured = []
-        topics = []
-
-        if step == 2 and not deadend:
-            featured = get_featured_articles(product)
-            topics = topics_for(product, parent=None)
-        elif step == 3:
-            form = NewQuestionForm(
-                product=product_config,
-                initial={"category": category_key},
-            )
-
-        return render(
-            request,
-            template,
-            {
-                "form": form,
-                "products": config.products,
-                "current_product": product_config,
-                "current_html": html,
-                "current_step": step,
-                "deadend": deadend,
-                "host": Site.objects.get_current().domain,
-                "featured": featured,
-                "topics": topics
-            },
+    form = None
+    images = None
+    featured = []
+    topics = []
+
+    if step == 2:
+        featured = get_featured_articles(product)
+        topics = topics_for(product, parent=None)
```

Comment:
Why don't you query here the db to get the topics for this product?

Reply:
it seemed easier to use this helper function